cc/private/toolchain/unix_cc_configure.bzl

```diff
@@ -666,6 +666,7 @@ def configure_unix_toolchain(repository_ctx, cpu_value, overridden_tools):
             "%{coverage_link_flags}": coverage_link_flags,
             "%{cxx_builtin_include_directories}": get_starlark_list(builtin_include_directories),
             "%{cxx_flags}": get_starlark_list(cxx_opts + _escaped_cplus_include_paths(repository_ctx)),
+            "%{fastbuild_compile_flags}": get_starlark_list(["-gmlt"]),
```

Comment:
-gmlt doesn't exist for all compilers. (I think that's the failure we're seeing in the presubmit).

Conservatively we probably don't want to alter the behavior of any existing toolchains in this PR, so this should be an empty list to start out?